doc/processing_publishing_api_messages.md

```diff
@@ -0,0 +1,39 @@
+# Processing messages from the PublishingAPI
```

Comment:
I think "PublishingAPI" is normally written "Publishing API", at least in it's own README https://github.com/alphagov/publishing-api#publishing-api